lively.ide/styling/shared.cp.js

```diff
@@ -26,7 +26,7 @@ const CloseButtonHovered = component(CloseButtonDefault, { name: 'close button h
 
 const CloseButton = component(CloseButtonDefault, {
   name: 'close button',
-  master: { auto: CloseButtonDefault, hover: CloseButtonHovered }
+  master: { hover: CloseButtonHovered }
```

Comment:
Previously that would lead to problems yes.
This was because we did not differentiate between the parent and the potentially overridden master in a policy. You could just have either but not both. Since we are now able to distinguish these two, we can more conveniently adjust the master like this because we *know* we can default to the parent.
I think that's the way it should be.